rust/processor/src/db/postgres/migrations/2024-09-24-061425_market_discovery/up.sql

```diff
@@ -0,0 +1,1174 @@
+-- Your SQL goes here
+CREATE TABLE unregistered_markets (
+    emojis BYTEA
+);
+
+CREATE TABLE emojis (
+    emoji BYTEA PRIMARY KEY
+);
+
+INSERT INTO emojis VALUES
```

Comment:
- [ ] Include description either here or in README on how to generate this

Reply:
WDYM on how to generate this ? The user is not supposed to generate anything, everything happens in the migration.

Reply:
> WDYM on how to generate this ?

E.g. how did you manipulate text to generate the list of viable emojis, since ultimately `hex_codes.move` is the source of truth. Basically showing how this file is generated. I've captured this in a follow-up task ECO-2218 so that merge isn't blocked